koap/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ kotlin {
     jvm()
     js().browser()
     macosX64()
+    macosArm64()
```

Comment:
Wow, hard to believe it can be this simple when there's no platform specific source sets